app_dart/bin/server.dart

```diff
@@ -216,6 +216,33 @@ Future<void> main() async {
         delegate: GetStatus(config),
       ),
 
+      /// Returns a list of the commit shas that had green runs.
+      ///
+      /// A green commit is used to help the release tooling find commits Flutter infrastructure has validated. The rules are:
+      /// 1. A commit had all its tests run (at least those that are not in bringup)
+      /// 2. All those blocking tasks were green
+      /// Green commit shas are returned in the order of commit timestamp.
+      ///
+      /// Parameters:
+      ///   repo: default: 'flutter'. Name of the repository.
+      ///
+      /// GET: /api/public/get-green-commits?repo=$repo
+      ///
+      /// Response: Status: 200 OK
```

Comment:
I would avoid hard coding a response here, but instead say it's a List<String> of the commit shas

Reply:
umm Maybe my understanding is not correct, but I think the get() methods in request handlers override the Future<Body> get() function, and returns a Body type, which from my understanding represents a HTTP response body. Correct me if I am wrong, I think if we use a HTTP get request and it succeeds with a response body, my understanding is that we are not hard coding it?

Reply:
You're hard coding an example response in the documentation. Prefer giving the type returned instead as it's more concise and obvious to update when we change the logic here

Reply:
I see. My bad, I misunderstood it as my response was hard coded instead of generated.